### PR TITLE
Generalize Etherscan tutorial (somewhat)

### LINKED ISSUE #1667: Generalize Etherscan tutorial
- [x] Make language general so it applies to any EMP token
- [x] Indicate that the pictures reflect ETHBTC
- [x] Mention `xyCOMP` up front so people know they're on the right page

### PR DESCRIPTION
**Pull request by adrianmcli**
Fixes #1667 

I've modified the tutorial to become a little more general and mention `xyCOMP` upfront. Since the name is still kind of yet to be determined, I've just decided to call it `xyCOMP` for now.

We don't yet have a deployed address for `xyCOMP` yet, so I haven't put those in. But the way I've re-written some of sentences make it a lot more obvious and easier to slot it in once it's ready.

### CODE REVIEW

File: documentation/synthetic_tokens/tutorials/mint_tokens_via_etherscan.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-## Before we begin: DAI approval
+## Before we begin: collateral approval
 
-The ETHBTC token minting contract needs approval to transfer DAI (the collateral currency) on your behalf. To do that, follow these instructions:
+The token minting contract needs approval to transfer the collateral currency (DAI in this case) on your behalf. To do that, follow these instructions:
 
 1. Go to the [Write Contract Tab](https://etherscan.io/address/0x6b175474e89094c44da98b954eedeac495271d0f#writeContract) on the [DAI](https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f) contract page.
 2. Search for the `approve` function.
-3. For the first argument, pass in the address of the ETHBTC token minting contract: `0x3f2d9edd9702909cf1f8c4237b7c4c5931f9c944`.
+3. For the first argument, pass in the address of the token minting contract (for ETHBTC on mainnet it is `0x3f2d9edd9702909cf1f8c4237b7c4c5931f9c944`).
```
Comment (mrice32) [inline documentation/synthetic_tokens/tutorials/mint_tokens_via_etherscan.md:13]: nit: thoughts on having an appendix or some other page where we keep a curated list of deployed EMP addresses?

Comment (adrianmcli) [reply documentation/synthetic_tokens/tutorials/mint_tokens_via_etherscan.md:13]: I think in the short-to-medium term we can manage a page like that in the docs, but I think it makes more sense when there are three or four EMPs.

In the long term, I'd like to see a frontend that lists all existing EMPs in the Registry.

Comment (mrice32) [reply documentation/synthetic_tokens/tutorials/mint_tokens_via_etherscan.md:13]: 100% agreed. A dynamic dashboard would be the best here. SGTM -- we can put off creating a list until we have a few more EMPs.

**Comment by adrianmcli** [comment]
I've renamed `xyCOMP` to `yCOMP`. I think this is ready to be merged, unless we want to wait till we have a deployed address for `yCOMP`. Thoughts @mrice32?

**Pull request merged**